java/java-hello-world/pom.xml

```diff
@@ -37,13 +50,24 @@
 			<artifactId>spring-webmvc</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-thymeleaf</artifactId>
+		</dependency>
+
 		<dependency>
```

Comment:
nit: it is nice to have test dependencies listed last. Also the formatting/indentation of this file is a little off.

Reply:
Fixed